scripts/lib/mkcloud-libvirt.sh

```diff
@@ -306,11 +306,9 @@ function libvirt_do_onhost_deploy_image()
     local disk=$3
 
     [[ $clouddata ]] || complain 108 "clouddata IP not set - is DNS broken?"
-    pushd /tmp
-    local wgetcachemode=-N
-    [[ $want_cached_images = 1 ]] && wgetcachemode=-nc
```

Comment:
There are more external users of mkcloud than our remote workers. Some of our partners that use it to deploy clouds for testing/development. They prefer to put the image on the host manually and have a switch to skip any downloading attempts (no wget, not rsync, because they have no connection to the outside).

So we need to maintain the switch to skip the downloading entirely and only complain if the image is missing.
Thats what my proposal is about.